riva/clients/asr/riva_asr_client.cc

```diff
@@ -401,6 +403,7 @@ class RecognizeClient {
   bool automatic_punctuation_;
   bool separate_recognition_per_channel_;
   bool speaker_diarization_;
+  int32_t max_speaker_count_;
```

Comment:
`max_speaker_count_` can also be `diarization_max_speakers_` to use same name all through